docs/visualize/tilemap.asciidoc

```diff
@@ -77,6 +77,10 @@ based on the geohash coordinates.
 
 NOTE: By default, the *Change precision on map zoom* box is checked. Uncheck the box to disable this behavior.
 
+NOTE: By default, the *place markers off grid (use geocentroid) box is checked. When this box is checked, the markers are
```

Comment:
Is there a reason why the two configuration options are notes and not called out explicitly in either a paragraph or a title / description approach? The precision slider is also a configuration and it is part of a dedicated paragraph. Maybe there is a better way to represent the two configuration options here.